src/lib/constants.ts

```diff
@@ -1,4 +0,0 @@
-export default {
-  barHeight: 48,
```

Comment:
I got rid of `lib/constants.ts` since these are just used in a single place now as default props

But if you still prefer this pattern I can refactor back to fetching the default prop values from here?